rpcq/test/test_rpc.py

```diff
@@ -122,6 +129,13 @@ def test_client_rpcerrorerror(server, client):
         assert 'ValueError: Oops.' in full_traceback
 
 
+def test_client_warning(server, client):
+    with catch_warnings(record=True) as warnings:
+        result = client.call('just_a_warning')
+        assert result == 5
+        assert len(warnings) > 0
```

Comment:
Add test here that the warning string matches and that the type is identified

Reply:
nice, this was good advice